Wed Sep 25 15:08:33 2024       
+-----------------------------------------------------------------------------+
| NVIDIA-SMI 470.161.03   Driver Version: 470.161.03   CUDA Version: 11.4     |
|-------------------------------+----------------------+----------------------+
| GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
|                               |                      |               MIG M. |
|===============================+======================+======================|
|   0  Tesla K80           Off  | 00000000:04:00.0 Off |                    0 |
| N/A   61C    P0   146W / 149W |    573MiB / 11441MiB |     98%      Default |
|                               |                      |                  N/A |
+-------------------------------+----------------------+----------------------+
|   1  Tesla K80           Off  | 00000000:05:00.0 Off |                    0 |
| N/A   45C    P0   148W / 149W |    573MiB / 11441MiB |     98%      Default |
|                               |                      |                  N/A |
+-------------------------------+----------------------+----------------------+
|   2  Tesla K80           Off  | 00000000:83:00.0 Off |                    0 |
| N/A   46C    P0   144W / 149W |    660MiB / 11441MiB |     98%      Default |
|                               |                      |                  N/A |
+-------------------------------+----------------------+----------------------+
|   3  Tesla K80           Off  | 00000000:84:00.0 Off |                    0 |
| N/A   42C    P0    74W / 149W |      0MiB / 11441MiB |     97%      Default |
|                               |                      |                  N/A |
+-------------------------------+----------------------+----------------------+
                                                                               
+-----------------------------------------------------------------------------+
| Processes:                                                                  |
|  GPU   GI   CI        PID   Type   Process name                  GPU Memory |
|        ID   ID                                                   Usage      |
|=============================================================================|
|    0   N/A  N/A     18758      C   ...ps/amber22/bin/pmemd.cuda      570MiB |
|    1   N/A  N/A     18302      C   ...ps/amber22/bin/pmemd.cuda      570MiB |
|    2   N/A  N/A     19560      C   ...ps/amber22/bin/pmemd.cuda      657MiB |
+-----------------------------------------------------------------------------+
{'checkpoint': 'results/cifar100/RPS_CIFAR_M8_J1', 'labels_data': 'prepare/cifar100_10.pkl', 'savepoint': '', 'num_class': 100, 'class_per_task': 10, 'M': 8, 'jump': 2, 'rigidness_coff': 2.5, 'dataset': 'CIFAR', 'epochs': 100, 'L': 9, 'N': 1, 'lr': 0.001, 'train_batch': 128, 'test_batch': 128, 'workers': 16, 'resume': False, 'arch': 'res-18', 'start_epoch': 0, 'evaluate': False, 'sess': 0, 'test_case': 0, 'schedule': [20, 40, 60, 80], 'gamma': 0.5}
Traceback (most recent call last):
  File "cifar.py", line 219, in <module>
    main()
  File "cifar.py", line 89, in main
    model = RPS_net_cifar(args).cuda() 
  File "/home/pricej18/rpsnet/RPSnet-master/rps_net.py", line 229, in __init__
    self.init(None)
  File "/home/pricej18/rpsnet/RPSnet-master/rps_net.py", line 322, in init
    self.final_layer1 = nn.Linear(a5, args.num_class)
NameError: name 'args' is not defined
{'checkpoint': 'results/cifar100/RPS_CIFAR_M8_J1', 'labels_data': 'prepare/cifar100_10.pkl', 'savepoint': '', 'num_class': 100, 'class_per_task': 10, 'M': 8, 'jump': 2, 'rigidness_coff': 2.5, 'dataset': 'CIFAR', 'epochs': 100, 'L': 9, 'N': 1, 'lr': 0.001, 'train_batch': 128, 'test_batch': 128, 'workers': 16, 'resume': False, 'arch': 'res-18', 'start_epoch': 0, 'evaluate': False, 'sess': 0, 'test_case': 0, 'schedule': [20, 40, 60, 80], 'gamma': 0.5}
Traceback (most recent call last):
  File "cifar.py", line 219, in <module>
    main()
  File "cifar.py", line 89, in main
    model = RPS_net_cifar(args).cuda() 
  File "/home/pricej18/rpsnet/RPSnet-master/rps_net.py", line 229, in __init__
    self.init(None)
  File "/home/pricej18/rpsnet/RPSnet-master/rps_net.py", line 322, in init
    self.final_layer1 = nn.Linear(a5, args.num_class)
NameError: name 'args' is not defined
{'checkpoint': 'results/cifar100/RPS_CIFAR_M8_J1', 'labels_data': 'prepare/cifar100_10.pkl', 'savepoint': '', 'num_class': 100, 'class_per_task': 10, 'M': 8, 'jump': 2, 'rigidness_coff': 2.5, 'dataset': 'CIFAR', 'epochs': 100, 'L': 9, 'N': 1, 'lr': 0.001, 'train_batch': 128, 'test_batch': 128, 'workers': 16, 'resume': False, 'arch': 'res-18', 'start_epoch': 0, 'evaluate': False, 'sess': 0, 'test_case': 0, 'schedule': [20, 40, 60, 80], 'gamma': 0.5}
Traceback (most recent call last):
  File "cifar.py", line 219, in <module>
    main()
  File "cifar.py", line 89, in main
    model = RPS_net_cifar(args).cuda() 
  File "/home/pricej18/rpsnet/RPSnet-master/rps_net.py", line 229, in __init__
    self.init(None)
  File "/home/pricej18/rpsnet/RPSnet-master/rps_net.py", line 322, in init
    self.final_layer1 = nn.Linear(a5, args.num_class)
NameError: name 'args' is not defined
{'checkpoint': 'results/cifar100/RPS_CIFAR_M8_J1', 'labels_data': 'prepare/cifar100_10.pkl', 'savepoint': '', 'num_class': 100, 'class_per_task': 10, 'M': 8, 'jump': 2, 'rigidness_coff': 2.5, 'dataset': 'CIFAR', 'epochs': 100, 'L': 9, 'N': 1, 'lr': 0.001, 'train_batch': 128, 'test_batch': 128, 'workers': 16, 'resume': False, 'arch': 'res-18', 'start_epoch': 0, 'evaluate': False, 'sess': 0, 'test_case': 0, 'schedule': [20, 40, 60, 80], 'gamma': 0.5}
Traceback (most recent call last):
  File "cifar.py", line 219, in <module>
    main()
  File "cifar.py", line 89, in main
    model = RPS_net_cifar(args).cuda() 
  File "/home/pricej18/rpsnet/RPSnet-master/rps_net.py", line 229, in __init__
    self.init(None)
  File "/home/pricej18/rpsnet/RPSnet-master/rps_net.py", line 322, in init
    self.final_layer1 = nn.Linear(a5, args.num_class)
NameError: name 'args' is not defined
{'checkpoint': 'results/cifar100/RPS_CIFAR_M8_J1', 'labels_data': 'prepare/cifar100_10.pkl', 'savepoint': '', 'num_class': 100, 'class_per_task': 10, 'M': 8, 'jump': 2, 'rigidness_coff': 2.5, 'dataset': 'CIFAR', 'epochs': 100, 'L': 9, 'N': 1, 'lr': 0.001, 'train_batch': 128, 'test_batch': 128, 'workers': 16, 'resume': False, 'arch': 'res-18', 'start_epoch': 0, 'evaluate': False, 'sess': 0, 'test_case': 0, 'schedule': [20, 40, 60, 80], 'gamma': 0.5}
Traceback (most recent call last):
  File "cifar.py", line 219, in <module>
    main()
  File "cifar.py", line 89, in main
    model = RPS_net_cifar(args).cuda() 
  File "/home/pricej18/rpsnet/RPSnet-master/rps_net.py", line 229, in __init__
    self.init(None)
  File "/home/pricej18/rpsnet/RPSnet-master/rps_net.py", line 322, in init
    self.final_layer1 = nn.Linear(a5, args.num_class)
NameError: name 'args' is not defined
{'checkpoint': 'results/cifar100/RPS_CIFAR_M8_J1', 'labels_data': 'prepare/cifar100_10.pkl', 'savepoint': '', 'num_class': 100, 'class_per_task': 10, 'M': 8, 'jump': 2, 'rigidness_coff': 2.5, 'dataset': 'CIFAR', 'epochs': 100, 'L': 9, 'N': 1, 'lr': 0.001, 'train_batch': 128, 'test_batch': 128, 'workers': 16, 'resume': False, 'arch': 'res-18', 'start_epoch': 0, 'evaluate': False, 'sess': 0, 'test_case': 0, 'schedule': [20, 40, 60, 80], 'gamma': 0.5}
Traceback (most recent call last):
  File "cifar.py", line 219, in <module>
    main()
  File "cifar.py", line 89, in main
    model = RPS_net_cifar(args).cuda() 
  File "/home/pricej18/rpsnet/RPSnet-master/rps_net.py", line 229, in __init__
    self.init(None)
  File "/home/pricej18/rpsnet/RPSnet-master/rps_net.py", line 322, in init
    self.final_layer1 = nn.Linear(a5, args.num_class)
NameError: name 'args' is not defined
{'checkpoint': 'results/cifar100/RPS_CIFAR_M8_J1', 'labels_data': 'prepare/cifar100_10.pkl', 'savepoint': '', 'num_class': 100, 'class_per_task': 10, 'M': 8, 'jump': 2, 'rigidness_coff': 2.5, 'dataset': 'CIFAR', 'epochs': 100, 'L': 9, 'N': 1, 'lr': 0.001, 'train_batch': 128, 'test_batch': 128, 'workers': 16, 'resume': False, 'arch': 'res-18', 'start_epoch': 0, 'evaluate': False, 'sess': 0, 'test_case': 0, 'schedule': [20, 40, 60, 80], 'gamma': 0.5}
Traceback (most recent call last):
  File "cifar.py", line 219, in <module>
    main()
  File "cifar.py", line 89, in main
    model = RPS_net_cifar(args).cuda() 
  File "/home/pricej18/rpsnet/RPSnet-master/rps_net.py", line 229, in __init__
    self.init(None)
  File "/home/pricej18/rpsnet/RPSnet-master/rps_net.py", line 322, in init
    self.final_layer1 = nn.Linear(a5, args.num_class)
NameError: name 'args' is not defined
{'checkpoint': 'results/cifar100/RPS_CIFAR_M8_J1', 'labels_data': 'prepare/cifar100_10.pkl', 'savepoint': '', 'num_class': 100, 'class_per_task': 10, 'M': 8, 'jump': 2, 'rigidness_coff': 2.5, 'dataset': 'CIFAR', 'epochs': 100, 'L': 9, 'N': 1, 'lr': 0.001, 'train_batch': 128, 'test_batch': 128, 'workers': 16, 'resume': False, 'arch': 'res-18', 'start_epoch': 0, 'evaluate': False, 'sess': 0, 'test_case': 0, 'schedule': [20, 40, 60, 80], 'gamma': 0.5}
Traceback (most recent call last):
  File "cifar.py", line 219, in <module>
    main()
  File "cifar.py", line 89, in main
    model = RPS_net_cifar(args).cuda() 
  File "/home/pricej18/rpsnet/RPSnet-master/rps_net.py", line 229, in __init__
    self.init(None)
  File "/home/pricej18/rpsnet/RPSnet-master/rps_net.py", line 322, in init
    self.final_layer1 = nn.Linear(a5, args.num_class)
NameError: name 'args' is not defined
{'checkpoint': 'results/cifar100/RPS_CIFAR_M8_J1', 'labels_data': 'prepare/cifar100_10.pkl', 'savepoint': '', 'num_class': 100, 'class_per_task': 10, 'M': 8, 'jump': 2, 'rigidness_coff': 2.5, 'dataset': 'CIFAR', 'epochs': 100, 'L': 9, 'N': 1, 'lr': 0.001, 'train_batch': 128, 'test_batch': 128, 'workers': 16, 'resume': False, 'arch': 'res-18', 'start_epoch': 0, 'evaluate': False, 'sess': 0, 'test_case': 0, 'schedule': [20, 40, 60, 80], 'gamma': 0.5}
Traceback (most recent call last):
  File "cifar.py", line 219, in <module>
    main()
  File "cifar.py", line 89, in main
    model = RPS_net_cifar(args).cuda() 
  File "/home/pricej18/rpsnet/RPSnet-master/rps_net.py", line 229, in __init__
    self.init(None)
  File "/home/pricej18/rpsnet/RPSnet-master/rps_net.py", line 322, in init
    self.final_layer1 = nn.Linear(a5, args.num_class)
NameError: name 'args' is not defined
Done CODE
